lib/gen_stage.ex

```diff
@@ -661,7 +661,7 @@ defmodule GenStage do
   that the consumers have received it.
 
   The given message will be delivered in the format
-  `{{producer_pid, subscription_ref, msg}`, where `msg` is the message
+  `{producer_pid, subscription_ref, msg}`, where `msg` is the message
```

Comment:
The format should be `{{producer_pid, subscription}, msg}`. Can you please fix both accordingly? Thank you!